root/programs/C#/Frameworks/Infrastructure/Public/IO/SymmetricCryptography.cs

```diff
@@ -438,9 +438,9 @@ private static SymmetricAlgorithm CreateSymmetricAlgorithm(EnumSymmetricAlgorith
         {
             SymmetricAlgorithm sa = null;
 
-            // AesCryptoServiceProvider, AesManagedは3.5からの提供。
-            // 暗号化プロバイダ選択のポリシー（Managed → CSP → CNG）
-            // に合わせて、AesCryptoServiceProviderを削除。
+            // AesCryptoServiceProvider, AesManagedは.NET Framework 3.5からの提供。
+            // 暗号化プロバイダ選択の優先順は、Managed → CAPI(CSP) → CNG。
```

Comment:
@yusukemaegawa 

I've written a verification code for electronic signature processing with TestEncAndDecProvider.
Https://github.com/OpenTouryoProject/OpenTouryo/blob/MaintainingEncLib/root/programs/C%23/Frameworks/Tools/Encryption/TestEncAndDecProvider/Form1.cs#L1427

However, changes to the CNG library did not work just by replacing classes, as the interface change was great.